src/main/java/service/impl/ParserOperationImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.TransactionDto;
+import service.ParserOperation;
+
+public class ParserOperationImpl implements ParserOperation {
```

Comment:
not only the operation
```suggestion
public class TransactionParserImpl implements TransactionParser {
```